ci_framework/roles/set_openstack_containers/tasks/main.yml

```diff
@@ -106,6 +106,10 @@
   ansible.builtin.set_fact:
     operator_pod_name: "{{ '\n'.join(operator_pod_name.stdout_lines) |regex_search('^pod/' ~ operator_name ~ '-operator-controller-manager-.*$', multiline=True) }}"
 
+- name: Print the operator pod name
```

Comment:
We are literally printing operator_pod_name as string.


2023-07-13 09:14:14.686352 | controller | TASK [set_openstack_containers : Print the operator pod name msg=operator_pod_name] ***
2023-07-13 09:14:14.686358 | controller | Thursday 13 July 2023  09:14:14 -0400 (0:00:00.036)       0:18:23.420 *********
2023-07-13 09:14:14.725956 | controller | ok: [localhost] => {
2023-07-13 09:14:14.726022 | controller |     "msg": "operator_pod_name"
2023-07-13 09:14:14.726028 | controller | }

Reply:
done